### PR TITLE
Fixed the serializer handling for parameters.

### PR DESCRIPTION
**Pull request by brunobord**
They were parsed and checked, but not saved using this code.

### CODE REVIEW

**Comment by brunobord** [comment]
You can expect a PR in `django-formidable` to reflect this change (in the dj-formidable documentation)

File: formidable_color_picker/serializers.py [hunk 1]
```diff
@@ -9,7 +8,6 @@
 class ColorPickerFieldSerializer(FieldSerializer):
 
     type_id = 'color_picker'
-    parameters = serializers.JSONField()
```
Comment (alexdashkov) [inline formidable_color_picker/serializers.py:12]: To be honest, I've not checked but it seems that this value is required(But maybe with a ```write_only=True``` argument ). Am I wrong? 

```python
parameters = serializers.JSONField(write_only=True)
```

Comment (brunobord) [reply formidable_color_picker/serializers.py:12]: it didn't complain when I removed it in fact. I've tested locally using my copy of django-formidable with this branch.

**Review by syldb — APPROVED** [review]
yolo

**Pull request merged**